diesel_cli/src/cli.rs

```diff
@@ -218,6 +218,13 @@ pub fn build_cli() -> App<'static, 'static> {
                 .multiple(true)
                 .number_of_values(1)
                 .help("A list of types to import for every table, separated by commas"),
+        )
+        .arg(
+            Arg::with_name("generate-custom-type-definitions")
```

Comment:
Let's name this the same as `generate_missing_sql_type_definitions` and make it a boolean option while defaulting it to `true`. Which means we have to name the option `--no-generate-missing-sql-type-definitions`